codegen/src/main/scala/akka/grpc/gen/Main.scala

```diff
@@ -85,4 +85,16 @@ object Main extends App {
 
   System.out.write(out.toByteArray)
   System.out.flush()
+
+  /**
+   * Generators to generate both the 'plain' akka-grpc code and the play-specific code that depends on it.
+   * In other build tools (Maven and sbt) these are passed to protoc separately, but since Gradle does the
+   * akka-grpc code generation in a single protoc invocation we need to combine those here:
+   */
+  object CombinedPlayJavaClientCodeGenerator extends PlayJavaClientCodeGenerator with JavaClientCodeGenerator
+  object CombinedPlayJavaServerCodeGenerator extends PlayJavaServerCodeGenerator with JavaServerCodeGenerator
+  object CombinedPlayJavaBothCodeGenerator extends PlayJavaClientCodeGenerator with JavaClientCodeGenerator with PlayJavaServerCodeGenerator with JavaServerCodeGenerator
+  object CombinedPlayScalaClientCodeGenerator extends PlayScalaClientCodeGenerator with ScalaClientCodeGenerator
+  object CombinedPlayScalaServerCodeGenerator extends PlayScalaServerCodeGenerator with ScalaServerCodeGenerator
+  object CombinedPlayScalaBothCodeGenerator extends PlayScalaClientCodeGenerator with ScalaClientCodeGenerator with PlayScalaServerCodeGenerator with ScalaServerCodeGenerator
 }
```

Comment:
should all those `override def name`?

Reply:
I'm not sure, as only one is executed here anyway - but can't hurt to be sure